src/csharp/Microsoft.Spark/Utils/UdfSerDe.cs

```diff
@@ -331,69 +250,15 @@ private static TypeData SerializeType(Type type)
             {
                 Name = type.FullName,
                 AssemblyName = type.Assembly.FullName,
-                ManifestModuleName = type.Assembly.ManifestModule.Name
+                AssemblyFileName = Path.GetFileName(type.Assembly.Location)
```

Comment:
As far as I remember, `type.Assembly.ManifestModule.Name` also provided the assembly name. No?

Reply:
Eric had some concerns regarding ManifestModule.Name.  It seems for our tests, it has provided the assemblyfilename, but as I wasn't certain it's always the case, I opted to go this route.